lib/src/icons/Carat-Down.scss

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+.nx-icon--carat-down {
+  color: #000000;
+  height: 8px;
+  width: 12px;
```

Comment:
I'm a little worried that this icon doesn't behave the same way as all of our other icons, e.g. controllable via `font-size` and looking correct in relation to surrounding text at the same font size

Reply:
> I'm a little worried that this icon doesn't behave the same way as all of our other icons, e.g. controllable via `font-size` and looking correct in relation to surrounding text at the same font size

Switched to `em`: https://github.com/sonatype/sonatype-react-shared-components/pull/55/commits/5c6c68604acbee2b127d74adafd508f40ee0d035

Reply:
> Switched to em: 5c6c686

There's more to what I meant than that.  Ideally I'd want it to be 1em tall, with appropriate whitespace above and below the glyph, and with a correct vertical-align so it can be used anywhere as easily as an FA icon.  For that matter, why aren't we using the FA icon here?

I'm working on this change, but it causes some surprises in regards to the flex layout, which I'm seeing if also apply to the FA icon.

Reply:
> There's more to what I meant than that. Ideally I'd want it to be 1em tall, with appropriate whitespace above and below the glyph, and with a correct vertical-align so it can be used anywhere as easily as an FA icon. For that matter, why aren't we using the FA icon here?

See f4f8eeef813d6794bc23d470e8af for my suggested change if do in fact want to stick with the custom icon and not FA here.

Reply:
> > There's more to what I meant than that. Ideally I'd want it to be 1em tall, with appropriate whitespace above and below the glyph, and with a correct vertical-align so it can be used anywhere as easily as an FA icon. For that matter, why aren't we using the FA icon here?
> 
> See [f4f8eee](https://github.com/sonatype/sonatype-react-shared-components/commit/f4f8eeef813d6794bc23d470e8af9c8eddf83d41) for my suggested change if do in fact want to stick with the custom icon and not FA here.

As discussed replaced with FA icon https://github.com/sonatype/sonatype-react-shared-components/pull/55/commits/dce8a6f85281bf6b00383c9789a1b4b983ed030f